### ISSUE TITLE
INT_MIN is wrong in a few places

### ISSUE THREAD

**Issue by soggysec** [gnuradio/gnuradio#1791]
MAX_INT = (2^31)-1 // 2147483647
MIN_INT = (-MAX_INT - 1) // -2147483648

// Should replace all uses of the wrong value with -2147483648, OR better yet:

std::numeric_limits<int>::max()
std::numeric_limits<int>::min()
Uses of wrong value:

Note: The comment of MIN_INT is also incorrect, it is simplified to just -(2^31)
```
grep -rn "\-2147483647" . 
./gr-blocks/lib/float_array_to_int.cc:33:static const int64_t MIN_INT = -2147483647; // -(2^31)-1
```

**Comment by soggysec** [comment]
PR Here: 
https://github.com/gnuradio/gnuradio/pull/1792

**Comment by marcusmueller** [comment]
As discussed in the PR, I like your "OR better yet" approach the most :) Thanks for reporting and fixing this!!

**Comment by marcusmueller** [comment]
closed via #1792
